src/controllers/check.company.controller.ts

```diff
@@ -1,17 +1,17 @@
 import { NextFunction, Request, Response } from "express";
+import * as templatePaths from "../model/template.paths";
 
 export const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 
-    const scenario = { company: { PTFRequested: "Yes", accountsDue: "01 January 2020",
-            accountsStatus: "overdue", address: "25 No Street, Nowhere", companyType: "Private Limited Company",
-            csDue: "02 January 2020", csStatus: "overdue", incorporationDate: "01 January 2000",
-            name: "Test Company LTD", number: "00001111", status: "Active" }};
+  const scenario = { company: { PTFRequested: "Yes", accountsDue: "01 January 2020",
+    accountsStatus: "overdue", address: "25 No Street, Nowhere, nl2br", type: "Private Limited Company",
+    csDue: "02 January 2020", csStatus: "overdue", incorporationDate: "01 January 2000",
+    name: "Test Company LTD", number: "00001111", status: "Active" }};
 
-    return res.render("check-company", {
-        scenario,
-    });
+  return res.render(templatePaths.CHECK_COMPANY, {
+    scenario,
```

Comment:
We should remove references to scenario as they were used by the UX team to populate their prototype screens ie. test scenarios. So pass company instead and in the template scenario.company.stuff would become company.stuff.